src/de/events.md

```diff
@@ -0,0 +1,98 @@
+---
+outline: deep
+cSpell: locale de
+---
+
+# Events
+
+## Event erstellen
+
+Du kannst jederzeit ein Event für deine Community erstellen indem du das `+` Symbol neben deinem Avatar anklickst und **Event erstellen** auswählst.
```

Comment:
Vielleicht wär hier n Hint sinnvoll in dem nochmal klar gestellt wird, dass ein Event immer einer Community angehört. Das ist glaub ich nicht jedem sofort klar 🤔

Reply:
Ja, darüber hab ich auch schon nachgedacht. Werde ich nachreichen.